SU2_CFD/src/iteration_structure.cpp

```diff
@@ -494,95 +494,46 @@ void CMeanFlowIteration::Iterate(COutput *output,
                                  CFreeFormDefBox*** FFDBox,
```

Comment:
Just to make sure I understand. Here the inner loop for unsteady simulations has been removed, so now to iterate the solver, the loop has to be added in the Driver level, and iterate over IntIter. 

More generally, within each time step of the unsteady solver (i.e., within 1 "ExtIter"), there are a number of IntIters until converged, then we call CMeanFlowIteration::Update to move on in time (t_{n+1} <- t_n). This is very similar to the steady-state solver, although in that case the iteration loop is in ExtIter (which I find somehow confusing).

Am I right with this? Because in a separate branch I had put together a separate class to steady-state FSI problems, and that will need some refurbishing before being pushed to develop (although I think it will look clearer this way).